cockroachcloud/export-logs.md

```diff
@@ -199,8 +201,8 @@ Perform the following steps to enable log export from your {{ site.data.products
 
 1. Add your {{ site.data.products.dedicated }} cluster's service account principal to the role you just created.
 
-	1. In the GCP console, visit the [IAM service accounts page](https://console.cloud.google.com/iam-admin/serviceaccounts) for your project.
- 1. Click the **+ Add** button.
+	1. In the GCP console, visit the [IAM admin page](https://console.cloud.google.com/iam-admin) for your project.
+ 1. Click the **+ Grant Access** button.
  1. In the box labeled **New principals**, enter the service account ID for the {{ site.data.products.dedicated }}-managed service account to which you will grant access. The service account ID takes the following format:
```

Comment:
```suggestion
 1. In the box labeled **New principals**, enter the service account ID for the {{ site.data.products.dedicated }}-managed service account to which you will grant access. The service account ID takes the following format:
```

Why `CockroachDB dedicated-managed service account`? Is this directly reflective of any names in the UI?

Reply:
Sure: to differentiate from the non-dedicated service account you are using, i.e. the one that you can name anything you like. This one must be named exactly as indicated.

Users providing their own GCP service account instead of this specific one has in fact been a user complaint (from SREs).

But also see: https://github.com/cockroachdb/docs/pull/15725 where this step is refactored anyways; and which includes all the changes from this PR anyways (had to keep moving to keep engineering happy).

Reply:
Thanks for the context and sorry I missed your reply earlier. I think we could potentially drop the hyphen, but I see we have existing instances of this in the docs, so I'm happy to proceed as-is and take this to a follow-up style discussion.